app/helpers/results_helper.rb

```diff
@@ -9,19 +9,42 @@ def render_field_item(doc)
         doc[:document][doc[:field]].join(", ").html_safe
   end
 
+  # truncate abstract to 450 characters
+  def render_abstract(doc)
```

Comment:
@eshadatta Go ahead and hardcode for now unless you can think of a better place to put it. We can always iterate if it comes to multiple long text fields with different limits but I don't see a use case for that yet. Tx